### ISSUE TITLE
best guess about given gender for name matching

### ISSUE THREAD

**Issue by jernsthausen** [datamade/dedupe-api#115]
Seems like it could be worth our while to take advantage of the gendering of most western names when matching records once probablepeople is incorporated into dedupe. Using something like a confidence score about gender could make a better model, but one might want to have that as an optional setting. I could imagine it not being useful if the entities that one is interest in are households.

Of course, this is not trivial, because we'll also need to be able to separate a name from entities of other kinds. Not sure if that's best handled separately by the user before uploading data or by dedupe afterwards.

**Comment by fgregg** [comment]
Yeah, I can see this working well. See https://github.com/datamade/probablepeople/issues/9 for an approach on how to do this.
